Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -100,8 +108,14 @@ public enum ExitValue { SUCCESS = 0, PREPROCESSING_ERROR, DAFNY_ERROR, COMPILE_E
       Contract.Requires(cce.NonNullElements(args));
 
       var dafnyOptions = new DafnyOptions();
-      CommandLineArgumentsResult cliArgumentsResult = ProcessCommandLineArguments(dafnyOptions, args, out var dafnyFiles, out var otherFiles);
-      var driver = new DafnyDriver(dafnyOptions);
+      var action = DriverAction.Run;
+      if (args.Length > 0 && args[0] == "format") {
+        action = DriverAction.Format;
+        args = args.Skip(1).ToArray();
+      }
```

Comment:
Is there a reason this logic can't be handled within `ProcessCommandLineArguments` itself?